cmd/run.go

```diff
@@ -60,11 +60,11 @@ var runCommand = &cobra.Command{
 				volume = user.HomeDir + volume[1:]
 			}
 			dockerArgs = append(dockerArgs, "-v")
-			dockerArgs = append(dockerArgs, volume)
+			dockerArgs = append(dockerArgs, os.ExpandEnv(volume))
 		}
```

Comment:
I'm mostly using it with another commit not in this PR.
I have extended the environment to get a 'GIT_ROOT' fed with the result of `git rev-parse --show-toplevel`.
I bind volumes  `${GIT_ROOT}:${GIT_ROOT}`, working directory `${PWD}` and an environment variable of `TOP_DIR=${GIT_ROOT}`
It allows me to run the same tool in different repos, from any sub-directory in the repository